framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -584,6 +587,23 @@ private void printStoredMessages(CompilationUnitTree unit) {
         if (messageStore != null) {
             for (CheckerMessage msg : messageStore) {
                 super.printOrStoreMessage(msg.kind, msg.message, msg.source, unit);
+                printStackTrace(msg.trace);
+            }
+        }
+    }
+
+    /**
+     * Prints given stack trace of runtime exception if -AdumpOnErrors flag is passed as compilation
```

Comment:
`Output the given stack trace if the "dumpOnErrors" option is enabled.`

Reply:
I have made the changes in my latest commit